### DISCUSSION TITLE
Can Sling read XLSX files (what about the 'sheet' parameter) ?

### DISCUSSION THREAD

**Discussion by fabrice-etanchaud** [slingdata-io/sling-cli#214] [Q&A]
Hi !

There is a `sheet `parameter : 

https://docs.slingdata.io/sling-cli/run/configuration#options

Does that mean that sling can read XLSX files ?

Thank you!

**Comment by flarco** [comment]
Yes, `sling run --src-stream file:///path/to/my_excel.xlsx --src-options '{ sheet: "Sheet1!B1:H70" }' --stdout -l 10`

**Reply by flarco** [reply]
valid entries for `sheet`: 
- `Sheet1`
- `Sheet1!B:H`
- `Sheet1!B1:H70`
